src/core/api/clock.ts

```diff
@@ -84,6 +84,7 @@ export interface IStalledStatus {
   /** What started the player to stall. */
   reason : "seeking" | // Building buffer after seeking
            "not-ready" | // Building buffer after low readyState
+           "internal-seek" | // Seeking performed internally
```

Comment:
Maybe we should be a little less vague here:
  1. this is only when building buffer (like the previous two reasons)
  2. "Seeking performed internally" make me believe that it is made by the clock. While it is kind of true, maybe we could add something to tell that it is when the RxPlayer triggered the seek, not the API (maybe we could also update the comment for "seeking" to better explain the difference)